cmd/gaia/app/sim_test.go

```diff
@@ -109,17 +109,15 @@ func appStateFn(r *rand.Rand, accs []simulation.Account) json.RawMessage {
 	}
 	fmt.Printf("Selected randomly generated slashing parameters: %+v\n", slashingGenesis)
 	mintGenesis := mint.GenesisState{
-		Minter: mint.Minter{
-			InflationLastTime: time.Unix(0, 0),
-			Inflation:         sdk.NewDecWithPrec(int64(r.Intn(99)), 2),
-		},
-		Params: mint.Params{
-			MintDenom:           stakeTypes.DefaultBondDenom,
-			InflationRateChange: sdk.NewDecWithPrec(int64(r.Intn(99)), 2),
-			InflationMax:        sdk.NewDecWithPrec(20, 2),
-			InflationMin:        sdk.NewDecWithPrec(7, 2),
-			GoalBonded:          sdk.NewDecWithPrec(67, 2),
-		},
+		Minter: mint.InitialMinter(
+			sdk.NewDecWithPrec(int64(r.Intn(99)), 2)),
+		Params: mint.NewParams(
```

Comment:
Why was this changed from the struct? I have no idea what these values mean anymore unless I goto the package definition.

Reply:
yeah, as developers, we should have no problem with going to the package definition in a keystroke with golang (use `go-def`) - also the intention is to avoid bugs in the future (I've caught many bugs due this programming style of defining structs instead of using `NewXxx` methods) - see https://github.com/cosmos/cosmos-sdk/issues/2835